lighthouse-core/computed/metrics/cumulative-layout-shift.js

```diff
@@ -49,10 +53,22 @@ class CumulativeLayoutShift {
       throw new LHError(LHError.errors.LAYOUT_SHIFT_MISSING_DATA);
     }
 
+    // Chromium will set `had_recent_input` if there was recent user input, which
+    // skips shift events from contributing to CLS. This flag is also set when Lighthouse changes
+    // the emulation size. This consistently results in the first shift event always being ignored
+    // for CLS. Since we don't expect any user input, conditionally add the first shift event to
+    // CLS if it was ignored.
+    // See https://bugs.chromium.org/p/chromium/issues/detail?id=1094974.
+    if (firstLayoutShift.args.data && firstLayoutShift.args.data.had_recent_input) {
```

Comment:
I hate to add more complexity, but couldn't there be multiple layout shift events with had_recent_input in the first 500ms?

Reply:
dedupe https://github.com/GoogleChrome/lighthouse/pull/11079#discussion_r452361140